discosrv/discosrv.go

```diff
@@ -103,6 +105,7 @@ func ImAliveProbe(nodeAddr string, nodeType common.NodeType, dsAddr string) (Dis
 	return reply, e
 }
 
+// ImAlivePoll sends which Nodes are online
```

Comment:
You got the comments of these two the wrong way round. `ImAliveProbe` returns the list of RMs and GSs, `ImAlivePoll` only tells the `discosrv` that it's alive, the callers are not expected to do anything with the results.